src/main/java/org/myrobotlab/service/Tracking.java

```diff
@@ -255,6 +255,9 @@ public Stats publishStats() {
 
   public void disable() {
     state = TrackingState.IDLE;
+    for (String filter : trackingFilters) {
+      send(cv, "removeFilter", filter);

```

Comment:
why remove?  why not just disable the filters?

Reply:
hmm ... I agree

Reply:
updated

Reply:
This still says  "removeFilter" .. did you forget to check in some changes?

Reply:
yeah ... noticed i didn't get all the updates I thought I did - will take me a little to organize

Reply:
updated